app/views/questions/search.html.erb

```diff
@@ -0,0 +1,30 @@
+<div class="container container_body">
+  <% if @result.count == 0 %>
+      <div class="row">
+        <h3> NO Record Found</h3>
+  <% else %>
+    <% @result.each do |q| %>
```

Comment:
if the `@result` only contains objects of questions then it should be called `@questions`